src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -182,26 +205,32 @@
            [:div {:style {:padding "0.5em 0 1em 0"}
                   :data-test-id (config/when-debug "method-config-name")} (:name config)]))
         (create-section-header "Referenced Method")
-        (create-section [MethodDetailsViewer
-                         (merge {:ref "methodDetailsViewer"
-                                 :name (:methodName methodRepoMethod)
-                                 :namespace (:methodNamespace methodRepoMethod)
-                                 :snapshotId (:methodVersion methodRepoMethod)
-                                 :onSnapshotIdChange #(this :-load-new-method-template %)}
-                                (utils/restructure config methods editing? wdl-parse-error))])
+        (let [matching-methods (filter #(and (= methodNamespace (:namespace %)) (= methodName (:name %))) methods-response)
```

Comment:
here also: do you need the `filter` now that the endpoint returns the right things?